packages/example-site-next/next.config.js

```diff
@@ -1,3 +1,6 @@
 const { configureNext } = require("tinacms-doc-toolkit");
 
-module.exports = configureNext({})
+module.exports = configureNext({
+  basePath: '/tinacms-doc-toolkit',
```

Comment:
maybe we should only do this when we are deploying to gh-pages and not when in local dev?